index.html

```diff
@@ -1,100 +1,122 @@
-    <option value="16">Okreg wyborczy 16</option>
-    <option value="17">Okreg wyborczy 17</option>
-    <option value="18">Okreg wyborczy 18</option>
-    <option value="19">Okreg wyborczy 19</option>
-    <option value="20">Okreg wyborczy 20</option>
-    <option value="21">Okreg wyborczy 21</option>
-    <option value="22">Okreg wyborczy 22</option>
-    <option value="23">Okreg wyborczy 23</option>
-    <option value="24">Okreg wyborczy 24</option>
-    <option value="25">Okreg wyborczy 25</option>
-    <option value="26">Okreg wyborczy 26</option>
-    <option value="27">Okreg wyborczy 27</option>
-    <option value="28">Okreg wyborczy 28</option>
-    <option value="29">Okreg wyborczy 29</option>
-    <option value="30">Okreg wyborczy 30</option>
-    <option value="31">Okreg wyborczy 31</option>
-    <option value="32">Okreg wyborczy 32</option>
-    <option value="33">Okreg wyborczy 33</option>
-    <option value="34">Okreg wyborczy 34</option>
-    <option value="35">Okreg wyborczy 35</option>
-    <option value="36">Okreg wyborczy 36</option>
-    <option value="37">Okreg wyborczy 37</option>
-    <option value="38">Okreg wyborczy 38</option>
-    <option value="39">Okreg wyborczy 39</option>
-    <option value="40">Okreg wyborczy 40</option>
-    <option value="41">Okreg wyborczy 41</option>
-  </select>
-  <div id="wybranyokreg"></div>
-
-
-
-
-
- <!-- skrypt do odczytu plikow z excela znaleziony w necie -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.17.2/xlsx.full.min.js"></script>
-
-
-<script>
-document.getElementById('przyciskexcel').addEventListener('click', function() {
-document.getElementById('filei').click();
-});
-
-document.getElementById('filei').addEventListener('change', function() {
-  var file = this.files[0];
-  if (file) {
-  loadedFile = file;
-  alert('Plik Excel został załadowany i zapisany.');
-      }
-    });
-    
-//tutaj znajduje sie kod do wczytania danych z excela 
-    document.getElementById('select').addEventListener('change', function() {
-      var selectedOption = parseInt(this.value);
-      if (loadedFile) {
-        var reader = new FileReader();
-        reader.onload = function(e) {
-          var data = new Uint8Array(e.target.result);
-          var workbook = XLSX.read(data, { type: 'array' });
-          var worksheet = workbook.Sheets[workbook.SheetNames[0]];
-          var selectedCell = worksheet[XLSX.utils.encode_cell({ r: selectedOption, c: 5 })];
-          var cellValue = selectedCell ? selectedCell.v : 'Brak danych';
-          document.getElementById('wybranyokreg').textContent = "Wybrany okreg wyborczy zawiera " + cellValue + " wyborcow";
-        };
-
-        reader.readAsArrayBuffer(loadedFile);
-      } else {
-        alert('Proszę najpierw załadować plik Excel.');
-      }
-    });
-  </script>
+    <h1>liczba wyborcow w danych okregach wyborczych</h1>
+    <button id="przyciskcsv">Załaduj plik CSV</button> <br>

```

Comment:
Minor:

In general, I suggest using English for anything that the user does not see (variable names, comments, IDs).

```suggestion
    <button id="loadFileButton">Załaduj plik CSV</button> <br>
```